precompiles/outposts/stride/tx_test.go

```diff
@@ -36,18 +37,22 @@ func (s *PrecompileTestSuite) TestLiquidStake() {
 			},
 			200000,
 			true,
-			fmt.Sprintf(cmn.ErrInvalidNumberOfArgs, 4, 0),
+			fmt.Sprintf(cmn.ErrInvalidNumberOfArgs, 1, 0),
 		},
 		{
 			"fail - token not found",
 			func() []interface{} {
 				err := s.network.App.StakingKeeper.SetParams(s.network.GetContext(), stakingtypes.DefaultParams())
 				s.Require().NoError(err)
 				return []interface{}{
-					s.keyring.GetAddr(0),
-					common.HexToAddress("0x1FD55A1B9FC24967C4dB09C513C3BA0DFa7FF687"),
-					big.NewInt(1e18),
-					"stride1mdna37zrprxl7kn0rj4e58ndp084fzzwcxhrh2",
+					stride.AutopilotArgs{
+						ChannelID:       channelID,
+						Sender:          s.keyring.GetAddr(0),
+						Receiver:        s.keyring.GetAddr(0),
+						Token:           common.HexToAddress("0x1FD55A1B9FC24967C4dB09C513C3BA0DFa7FF687"),
```

Comment:
Why are this addresses hardcoded. At least it my by useful to have them as constants within the tests with a name that describes what they are